packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts

```diff
@@ -328,52 +327,30 @@ export const deleteCredentialHandler = async ({ ctx, input }: DeleteCredentialOp
     }
   }
 
-  // Validated that credential is user's above
-  await prisma.credential.delete({
-    where: {
-      id: id,
-    },
-  });
-
   // Backwards compatibility. Selected calendars cascade on delete when deleting a credential
   // If it's a calendar remove it from the SelectedCalendars
   if (credential.app?.categories.includes(AppCategories.calendar)) {
-    const selectedCalendars = await prisma.selectedCalendar.findMany({
```

Comment:
We were searching for selected calendars that belonged to the same type of integration as the one being deleted. So if there was a second instance of a calendar credential with selected calendars, this query would return those records.